lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -216,6 +220,12 @@ def extract_student(student, features):
                 (cohort.name for cohort in student.course_groups.all() if cohort.course_id == course_key),
                 "[unassigned]"
             )
+
+        if include_team_column:
+            student_dict['team'] = next(
+                (team.name for team in student.teams.all() if team.course_id == course_key),
+                "[unavailable]"
```

Comment:
No, we don't try to i18n grade reports. Since they are generated on the workers, they don't have the request context to determine how to i18n the report.